.gitignore

```diff
@@ -383,6 +383,13 @@ experimental/generator-dotnet-yeoman/node_modules
 
 # Allow the root-level /packages/ directory or it gets confused with NuGet directories
 !/[Pp]ackages/*
+!/tests/[Pp]ackages/*
```

Comment:
There are a couple duplicates of the same directory, maybe can just use `!/tests/**/[Pp]ackages/*`?